lib/video_info/providers/youtubeplaylist_scraper.rb

```diff
@@ -0,0 +1,23 @@
+require 'nokogiri'
+require 'open-uri'
+require 'open_uri_redirections'
+
+class VideoInfo
+  module Providers
+    module YoutubePlaylistScraper
+      def description
+	data.css('meta')[1].values[1]
+      end
+
+      private
+
+      def _set_data_from_api(api_url = _api_url)
+	Nokogiri::HTML(open(api_url, :allow_redirections => :safe))
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not -5) spaces for indentation.<br>Tab detected.